react-accordion-component/my-app/src/Accordion Components/Accordion.tsx

```diff
@@ -0,0 +1,44 @@
+import { useState } from 'react';
+
+type AccordionProps = {
+  topics: { uuid: string; topic: string; description: string }[];
+};
+
+export default function Accordion({ topics }: AccordionProps) {
+  const [expandTopic, setExpandTopic] = useState('');
+
+  function handleClick(uuid: string) {
+    if (expandTopic === uuid) {
+      setExpandTopic((prevExpandTopic) => {
```

Comment:
Why do you use the function version of the state setter? Just pass the new value:
`setExpandTopic('');`
`setExpandTopic(uuid);`

Reply:
Mentioned this in previous exercise, but I learned to understand the state setter better this way